pkg/apis/operator/v1alpha1/tektonconfig_validation.go

```diff
@@ -115,6 +115,8 @@ func (tc *TektonConfig) Validate(ctx context.Context) (errs *apis.FieldError) {
 
 	errs = errs.Also(tc.Spec.Pipeline.PipelineProperties.validate("spec.pipeline"))
 
+	errs = errs.Also(tc.Spec.Pipeline.Options.validate("spec.pipeline.options"))
```

Comment:
we should be doing here the config options one, and for others we should in respective valdations. I think we should validate this for all components

Reply:
fixed